src/CommandHandlers/JoinPrivateRoomHandler.cs

```diff
@@ -13,6 +15,27 @@ public override Task Handle(Client client, NetworkObject receivedObject)
         string roomName = p.Get<string>("rn") + "_" + p.Get<string>("0");
         Room room = Room.GetOrAdd(roomName, autoRemove: true);
         client.SetRoom(room);
+
+        if(client.Room != null)
```

Comment:
(after call two previous lines) this always should be not null (if I'm not wrong) ... when this is null?